### PR TITLE
[Requirements] Updating packaging to be >= version 22.0

### PR DESCRIPTION
**Pull request by TedThemistokleous**
This causes additional build failures with this package lower than 22.0

### Description

Update python packaging 

### Motivation and Context

build/CI/development impediment

### CODE REVIEW

**Comment by TedThemistokleous** [comment]
@cloudhan @ytaous  This came up when working out our docker containers for CI.

File: requirements.txt.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 coloredlogs
 flatbuffers
 numpy >= @Python_NumPy_VERSION@
-packaging
+packaging >=22.0
```
Comment (snnn) [inline requirements.txt.in:4]: This change seems unnecessary. This file is to list what are needed when install a prebuilt ONNX Runtime package. Does it have this requirement?

**Review by snnn — COMMENTED** [review]
To keep a record, we need more information why this change is needed.

**Comment by cloudhan** [comment]
@TedThemistokleous Can you elaborate a bit what the version update fix/workaound perticularly? If possible, can you crosslink some (maybe similiar) issue reports on github?

File: requirements.txt.in (hunk 1 above)
Comment (TedThemistokleous) [reply requirements.txt.in:4]: This arose when we created our own Docker containers for 5.4 and other items had requirements for packaging to be >= 22.0

**Comment by ytaous** [comment]
@TedThemistokleous - is this possible to keep it within ROCm 5.4? This change is global, so we want to avoid it if possible.

**Comment by TedThemistokleous** [comment]
The issue here is that building with a packaging version below 22 for the MIGraphX builds breaks and we're grabbing your requirements file before we do a build.  Sure, maybe I can update our docker and just close this out as a work around.

**Comment by ytaous** [comment]
I mean you can create a file near the pipeline, and name it specifically or using a subdir, say migraphx-addition/requirements, and have MIGraphX build to pick it up and overwrite existing one, would this work?

location example for rocm python packaging: e.g., onnxruntime/tools/ci_build/github/pai/requirements.txt

I wonder u can also leveraging the same file. 
@PeixuanZuo - any suggestion?

**Comment by PeixuanZuo** [comment]
Adding a new file may be better than changing the global requirement.txt. We have a dockerfile(dockerfiles/Dockerfile.migraphx) to guide how to build ort+migraphx, it uses dockerfiles/scripts/install_common_deps.sh to install python package, is it possible to add a new one like dockerfiles/scripts/install_migraphx_deps.sh in the same place?

**Comment by TedThemistokleous** [comment]
We did a change for this on our end so I dont think this PR is required anymore and its now part of install_prereqs.sh

https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/08360e83dbe62477a9d91f3cc9fe7ecd5a1e000d/tools/install_prereqs.sh#L60

Closing this out as this is now a non issue. Thanks everyone.

**Pull request closed without merging**